index.js

```diff
@@ -1218,6 +1238,18 @@ module.exports = new Map([
 		['T', 'Yes'],
 		['True', 'Yes']
 	])],
+	[undefined, new Map([
+		['N', 'No'],
+		['F', 'No'],
+		['False', 'No'],
+		['Y', 'Yes'],
+		['T', 'Yes'],
+		['True', 'Yes'],
+		['R', 'Rotated'],
+		['Tr', 'Transformed_Rotated'],
+		['Tu', 'Transformed_Upright'],
+		['U', 'Upright']
+	])],
```

Comment:
This probably needs some extra processing (note the `undefined`)

Reply:
Once merged/released/upgraded, https://github.com/mathiasbynens/unicode-property-aliases/pull/2 will fix this.

Reply:
Did you mean to link elsewhere with that last comment?

Reply:
No, since [the build script uses `unicode-property-aliases`](https://github.com/mathiasbynens/unicode-property-value-aliases/blob/a351a8bbd4e81ccc5f4bea9ee43df33f1deeb593/scripts/build.js#L5), it needs to be upgraded as well.

Reply:
Ah no, it it’s the other repo; nvm!